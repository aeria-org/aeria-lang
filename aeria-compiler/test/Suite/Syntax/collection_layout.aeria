collection CollectionLayoutTest {
  layout {
    name "grid"
    options {
      title prop1
      badge prop2
      picture prop3
      information prop4
      active prop5
      translateBadge false
    }
  }
}
