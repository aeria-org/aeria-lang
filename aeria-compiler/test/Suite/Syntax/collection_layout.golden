{
  "kind": "Program",
  "collections": [
    {
      "name": {
        "value": "CollectionLayoutTest",
        "kind": "CollectionName"
      },
      "layout": {
        "options": {
          "translateBadge": false,
          "title": {
            "value": "prop1",
            "kind": "PropertyName"
          },
          "picture": {
            "value": "prop3",
            "kind": "PropertyName"
          },
          "kind": "LayoutOptions",
          "information": {
            "value": "prop4",
            "kind": "PropertyName"
          },
          "badge": {
            "value": "prop2",
            "kind": "PropertyName"
          },
          "active": {
            "value": "prop5",
            "kind": "PropertyName"
          }
        },
        "name": "grid",
        "kind": "CollectionLayout"
      },
      "kind": "Collection"
    }
  ]
}
